index.js

```diff
@@ -44,3 +44,4 @@
  });
 
  module.exports = instance;
+ module.exports.RTT = nRFjprog.RTT;
```

Comment:
How about changing this to `module.exports.RTT = nRFjprog.RTT()`?

I'd prefer the module exporting a RTT singleton, instead of a RTT constructor.

Reply:
Agreed. Will fix.